### PR TITLE
[IMP] improvements for the spanish translation of enterprise.rst

### PR DESCRIPTION
**Pull request by Lau-Odoo**

### CODE REVIEW

**Comment by mart-e** [comment]
@Lau-Odoo where does this translation comes from?
I think we made the current translations with layers (cc @odony )

**Comment by Lau-Odoo** [comment]
> @Lau-Odoo where does this translation comes from?
> I think we made the current translations with layers (cc @odony )

It is the translation CCO sent me. So I supposed this is the one you're speaking about (the one from the layers).

**Comment by jcs-odoo** [comment]
@Lau-Odoo Is it still blocked?

**Comment by Lau-Odoo** [comment]
> @Lau-Odoo Is it still blocked?

Yes, still waiting for odo's approval :)

**Comment by jcs-odoo** [comment]
> > @Lau-Odoo Is it still blocked?
> 
> Yes, still waiting for odo's approval :)

I thought the blocking issue was a missing PDF ? 🤨

**Comment by Lau-Odoo** [comment]
> > > @Lau-Odoo Is it still blocked?
> > 
> > 
> > Yes, still waiting for odo's approval :)
> 
> I thought the blocking issue was a missing PDF ? 🤨

That problem is solved. But everything related to the kind of contracts this PR is about needs ODO's approval 🙂

File: content/legal.rst
Comment (jcs-odoo) [inline content/legal.rst:130]: @Lau-Odoo 

Unless I'm mistaken, when you put this PR back to draft, it's because the pdf file (see lines 105-107) is pinging a pdf of v8 of the ES agreement. While your RST modification is updating from 8.0 -> 9.0 .
Here is the external link, still v8 from 2018 https://www.odoo.com/documentation/14.0/odoo_enterprise_agreement_es.pdf

Also, these PDF files don't seem to be maintained from the documentation repository. @odony @mart-e do you know why it is so and where they can be updated? Who is creating this PDF? Or can we at least go forward and merge this PR even if the PDF isn't up to date?

Also, shouldn't this PR be targeting v13.0 rather than 14.0? If so, you'll have to close this PR and create a new Pull Request as rebasing and changing the target on the PR would ping all code owners of this repo and that would suck big time ^^

Thank you very much :)

**Review by odony — APPROVED** [review]
Thanks for the PR, was very good overall!

Some of the minor points that needed fixing:
- wrong date and reference version (should match the master EN version, not the date of the translation)
- some indentation issue for reference annotations
- some section subtitles were changed to bold
- section 6.1 had a missing bullet point in the first list
- sections 6.4 and 6.5 missed indentation for the Definition block + title
- CLA footnote in section 7.1 was missing

Fixed by 114e86d67550c749584434de48801510fbb0d8da, so we should be good to go

**Comment by odony** [comment]
@robodoo r+

**Comment by odony** [comment]
@robodoo r+

**Comment by odony** [comment]
@robodoo rebase-ff

**Comment by jcs-odoo** [comment]
Muchas gracias @odony

**Comment by Lau-Odoo** [comment]
Hi @odony, thanks for your review!

**Pull request closed without merging**
